source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -46,7 +47,12 @@ class FilterConfig {
         clear_route_cache_(config.clear_route_cache()),
         max_request_bytes_(config.with_request_body().max_request_bytes()),
         status_on_error_(toErrorCode(config.status_on_error().code())), local_info_(local_info),
-        scope_(scope), runtime_(runtime), http_context_(http_context), pool_(scope.symbolTable()),
+        scope_(scope), runtime_(runtime), http_context_(http_context),
+        filter_enabled_(config.has_filter_enabled()
```

Comment:
nit: optimally, the optional would be embedded inside the helper. One way of doing this would be to either pass a pointer to the field or nullptr, and then do the logic inside such that if nullptr it always returns true?

Reply:
I considered this while implementing the helper, but decided to go this route, because there are some cases for which the logic inside should return `false` if it's a `nullptr`, e.g., for `shadow_enabled`.